content/tracing/setup/first_class_dimensions.md

```diff
@@ -0,0 +1,151 @@
+---
+title: First-Class Dimensions
+kind: documentation
+aliases:
+  - /tracing/environments/
+  - /tracing/setup/environment
+further_reading:
+- link: "tracing/setup/docker"
+  tag: "Documentation"
+  text: Docker setup
+- link: "tracing/setup/go"
+  tag: "Documentation"
+  text: Go language instrumentation
+- link: "tracing/setup/java"
+  tag: "Documentation"
+  text: Java language instrumentation
+- link: "tracing/setup/python"
+  tag: "Documentation"
+  text: Python language instrumentation
+- link: "tracing/setup/ruby"
+  tag: "Documentation"
+  text: Ruby language instrumentation
+---
+
+There are several dimensions that you can configure to scope an entire Datadog APM application. This includes aggregate statistics (such as requests/second, latency, error rate, Apdex score) and visible traces.
+
+## Environment
+### Definition
+
+An environment is a first-class dimension that is used to scope an entire Datadog APM application. Some display settings are shared across environments, but all the measurable data (traces/metrics/statistics) can not be re-aggregated across multiple environments. Use cases include:
```

Comment:
An environment is a first-class dimension that is used to scope an entire Datadog APM application. Some display settings are shared across environments, but not all the measurable data (traces/metrics/statistics) can be re-aggregated across multiple environments. Use cases include: